services/blockchain-connector/shared/sdk/interoperability.js

```diff
@@ -40,16 +40,15 @@ const getChainAccount = async (chainID) => {
 };
 
 const getMainchainID = async () => {
+	const { chainID } = await getNodeInfo();
+
 	try {
 		if (!mainchainID) {
```

Comment:
Should we move getNodeInfo to under try so that we catch and throw timeout exception?

```suggestion
	try {
		const { chainID } = await getNodeInfo();
		
		if (!mainchainID) {
```